features/premium/subscribe_self_to_premium_mob.feature

```diff
@@ -47,3 +47,11 @@ Feature: Subscribe Self to Premium
     When Paypal updates our endpoint incorrectly
     Then "sam-buyer@agileventures.org" should not receive a "Welcome to AgileVentures Premium Mob" email
     And I should see "redirected" in last_response
+
+  Scenario: PayPals redirect doesn't work
+    Given I have logged in
+    And I visit "subscriptions/new?plan=premiummob"
+    Then I should see a paypal form within the paypal_section
+    When Paypal updates our endpoint for premium mob via get
+    Then "sam-buyer@agileventures.org" should receive a "Welcome to AgileVentures Premium Mob" email
+    And I should see "Thanks, you're now an AgileVentures Premium Mob Member!" in last_response
```

Comment:
Great work!! You might want to add a new line here

Reply:
great @faithngetich thanks :blush: